ERCS/erc-7699.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 7699
```

Comment:
```suggestion
eip: 7704
```
ERC number already assigned.  Please also update the filename.
https://github.com/ethereum/ERCs/pull/414/files#r1593721372